<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolSoundfontExport_gui</class>
 <widget class="QWidget" name="ToolSoundfontExport_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushTick">
       <property name="text">
        <string>Select all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushUntick">
       <property name="text">
        <string>Unselect all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="listPresets">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>120</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboFormat">
       <item>
        <property name="text">
         <string notr="true">.sf2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">.sf3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">.sfz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushFolder">
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBank">
       <property name="text">
        <string>one directory per bank</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QCheckBox" name="checkGM">
       <property name="text">
        <string>General MIDI classification</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="checkPreset">
       <property name="text">
        <string>preset number as prefix</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineFolder"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="comboQuality">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>High</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Low</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QCheckBox" name="checkFilePreset">
       <property name="text">
        <string>one file per preset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelQuality">
       <property name="text">
        <string>Quality</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushExport">
       <property name="text">
        <string>&amp;Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushAnnuler">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushTick</tabstop>
  <tabstop>pushUntick</tabstop>
  <tabstop>listPresets</tabstop>
  <tabstop>lineFolder</tabstop>
  <tabstop>comboFormat</tabstop>
  <tabstop>checkPreset</tabstop>
  <tabstop>checkBank</tabstop>
  <tabstop>checkGM</tabstop>
  <tabstop>pushExport</tabstop>
  <tabstop>pushAnnuler</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
